### ISSUE TITLE
V12 Support

### ISSUE THREAD

**Issue by grumpk1n** [Nachtrose/NRSAP#22]
I'm a big fan of this module and have used it hundreds of times for my v10 and v11 games.  Making the jump to v12, is this in the cards you think?

Thank you in advance.

**Comment by danimrath** [comment]
I don't know if they will support this module but I can drop some instructions here for anyone else who would like to fix their own.

Macro changes.
1) change references of loopingBackground.data.alpha to instead just be loopingBackground.alpha

2) For the sequencer macros change the following line
				canvas.scene.update({ img: img });
    to 
				canvas.scene.update({ "background.src": img });

3) For the with-tile macros change the following line
				coreBackground.update({ img: img });
    to 
                                coreBackground.update({ "texture.src": img });

4) On each scene double click the foreground tile that has the overlay and go to the overhead tab. Update the opacity setting slider to be 1.0 instead of 0.0 so that it stops dissappearing when you hover over it.

**Comment by grumpk1n** [comment]
Amazing, that fixed it pretty much.  Thank you very much.
The opacity of the foreground I'm not seeing though.  Do I just add that to the foreground select macro?

**Comment by danimrath** [comment]
Sorry I clarified in the list. You need to update the foreground tile itself in the overhead settings. There is an opacity slider. Just make sure it is the tile with the ship image and not the animation tile. They are both full screen so selecting the wrong one is easy to accidentally do.

**Comment by grumpk1n** [comment]
Ok that works.  Thank you.
Do you know how to fix the image from loading during the jump? Its like a loop and in between the loop the destination image flashes for a second, then resumes the loop.

**Comment by danimrath** [comment]
Yeah just use the scenes and macros that are for the "with tile" set. That happens because the other ones change the background image of the scene itself which forces it to reload. The with tiles scenes have tiles for both the foreground and background. Just make sure to update the macro like in my instructions.

**Comment by grumpk1n** [comment]
There does not appear to be a tile for the foreground (cockpit) image.  When I use the macro to select one, it runs, but does nothing.  Curious.  I fixed this before by setting the foreground to be the correct cockpit, but I can't seem to do that with the blue one.  I guess I'm married to the blue one.
